File: collection_1/accumulator.cpp
#include <cstdio>
using namespace std;

/**
 * https://dsa.cs.tsinghua.edu.cn/oj/problem.shtml?id=1053
 */
int main()
{
    // log2(10e8) ≈ 30
    setvbuf(stdin, new char[1 << 20], _IOFBF, 1 << 20);
    setvbuf(stdout, new char[1 << 20], _IOFBF, 1 << 20);
    int n;
    scanf("%d", &n);
    for (int i = 0; i < n; i++)
    {
        int a, b;
        scanf("%d%d", &a, &b);
        printf("%d\n", a + b);
    }
    return 0;
}

File: collection_1/accumulator.py
#!/usr/bin/env python3
# https://dsa.cs.tsinghua.edu.cn/oj/problem.shtml?id=1053
"""
demo:
    3
    10 30
    20 40
    15 22
Return
    40
    60
    37
"""

if __name__ == '__main__':
    n = int(input())
    s = []
    for i in range(n):
        int_arr = input().split()
        sum = int(int_arr[0])+int(int_arr[1])
        s.append(sum)
    print("\n".join(map(str,s)))
